cfgov/jinja2/v1/_includes/organisms/post-preview.html

```diff
@@ -60,7 +65,26 @@
                                     use_post_category) }}
         {% endif %}
     </div>
-    {% if settings.image_url %}
+    {% if settings.event_start_dt %}
+        <div class="o-post-preview_image-container">
+            <time class="calendar-icon"
+                data-month="{{ settings.event_start_dt | date('%b') }}"
+                data-day="{{ settings.event_start_dt | date('%-d') }}"
+                datetime="{{ settings.event_start_dt | date('%c') }}">
+                <span class="u-visually-hidden">{{ settings.event_start_dt| date('%b %-d, %Y') }}</span>
+            </time>
+            <img class="o-post-preview_image"
+                 src="{{ location_image_url({
+                            'location': settings.event_city
+                                        | default('Washington, DC', true)
+                                        | urlencode,
+                            'zoom':     '12',
+                            'scale':    '2',
+                            'size':     '276x155'
+                        }) }}"
+                 alt="Google Maps image of {{ settings.event_venue }}">
```

Comment:
Won't some events be online and not have an associated map? This is probably carryover from the existing codebase, but just checking.